test/data/realm.json

```diff
@@ -0,0 +1,10 @@
+{
+  "server_url": "https://satellite.redhat.local",
+  "username": "admin",
+  "password": "redhat",
+  "verify_ssl": false,
+  "name": "EXAMPLE.COM",
+  "realm_proxy": 1,
+  "realm_type": "Red Hat Identity Management",
+  "state": present,
```

Comment:
this should be quoted to be valid json, no?

Reply:
Fixed typo in   695d168..8113741  claudiol-org-realm-update -> claudiol-org-realm-update
 push.